profile/README.md

```diff
@@ -1,12 +1,15 @@
+![Fulcrum Genomics logo](https://avatars.githubusercontent.com/u/13546013?s=200&v=4)
```

Comment:
1. Lets shrink the logo using some html (see suggestion)
2. Lets add our full logo (with text).  Can you add it to the repo under `profile/assets/logo.svg`?  You can use the one here: https://github.com/nf-core/website/blob/main/src/assets/images/blog/fastquorum/fulcrumgenomics.svg

```suggestion
<a href="https://fulcrumgenomics.com/" alt="Fulcrum Genomics Logo">
<img src="https://avatars.githubusercontent.com/u/13546013?s=200&v=4" width="64">
</a>
```

Reply:
sorry, I misunderstood your request. I'm on it!